tests/integration_tests/test_defaults.py

```diff
@@ -0,0 +1,98 @@
+"""Tests here shouldn't require any sort of user data or instance setup."""
```

Comment:
I like this idea a lot. Common case tests like this have many benefits:

- fast (class_client)
- high-impact (add coverage that benefits most users)
- relatively easy to read/write (not covering some bespoke feature/cloud)